### DISCUSSION TITLE
Horizontal lines in gist embed

### DISCUSSION THREAD

**Discussion by Abufari** [jpanther/congo#219] [Q&A] [answered]
The gist embed works very good. But I dislike the horizontal lines between each line of code, it's quite distracting to me.
Most other sites I've seen don't show these lines from gist embeds. But as far as I can tell Hugo just embeds the gist javascript, so I don't know where the styling comes from. I found no other CSS files containing the word `gist`.

Example:
<img width="653" alt="Bildschirmfoto 2022-06-17 um 22 39 22" src="https://user-images.githubusercontent.com/8281170/174398204-bbb02091-8fb1-4b5e-92a1-e4294e454d78.png">

How can I disable the horizontal lines between lines of code?

**Comment by jpanther** [comment, marked as answer]
Thanks for the question. This behaviour is because the gist embed inserts its contents as a HTML table and Congo styles table rows with a bottom border. If you don't want this behaviour, add some [custom CSS](https://jpanther.github.io/congo/docs/advanced-customisation/#overriding-the-stylesheet) to your project to remove the border. Something like this should do the trick:

```css
.gist table tr {
    border-bottom: none;
}
```
